README.md

```diff
@@ -6,33 +6,103 @@ This plugin allows you to mount remote folder using sshfs in your container easi
 
 ## Usage
 
+### Use with SSH keys
 1 - Install the plugin
 
 ```
 $ docker plugin install vieux/sshfs # or docker plugin install vieux/sshfs DEBUG=1
+Plugin "vieux/sshfs" is requesting the following privileges:
+ - network: [host]
+ - mount: [/tmp]
+ - device: [/dev/fuse]
+ - capabilities: [CAP_SYS_ADMIN]
+Do you grant the above permissions? [y/N] y
+latest: Pulling from vieux/sshfs
+e248a6530152: Download complete
+Status: Downloaded newer image for vieux/sshfs:latest
+Installed plugin vieux/sshfs
+```
+
+2 - Configure driver to point to your SSH keys
+
+_NOTE_: The plugin defaults to looking for SSH keys in `/tmp`. You can copy your `<identity_file_name>` to `/tmp` and omit this step, e.g.
```

Comment:
Perhaps we should suggest they generate a new set of keys and add them to the `authorized_keys` file of the remote host, rather than copy?  Mainly because if they wanted to do this on a remote daemon, they probably don't want to copy their local keys over?  (Also, for clarity, we might want to specifically say "the plugin defaults to looking for SSH keys in `/tmp` on the daemon machine".)